src/components/rightSidebar/Users.vue

```diff
@@ -0,0 +1,79 @@
+<template xmlns:v-slot="http://www.w3.org/1999/XSL/Transform">
```

Comment:
why do you need `xmlns:v-slot="http://www.w3.org/1999/XSL/Transform"`?

Reply:
When I use `v-slot` directive, WebStorm highly recommends generating a namespace declaration. 
Reason: _Namespace 'v-slot' is not bound_

Reply:
I guess it's I bug in webstorm: https://youtrack.jetbrains.com/issue/WEB-37732